src/schema/rhcos.json

```diff
@@ -0,0 +1,160 @@
+{
+    "build-url": "https://example.org/job/rhcos-rhcos-4.4/5/",
```

Comment:
What is this file doing?

Reply:
I was going to add tests for the schema based on the current know-good meta.json from both RHCOS and FCOS.

Reply:
Hmm, maybe the file path should be e.g. `tests/fixtures/{rhcos,fcos}.json` or something? Doesn't feel like it belongs in `src/`.

We should also add schema validation to CI in `.cci.jenkinsfile`.

Reply:
I added a `meta --get name` to `.cci.jenkinsfile`. 

There is an implicit validation that happens, though, with anything that uses the `GenericBuildMeta`. However, I agree an explicit test is good.